File: 3. DSA/04. Python/01. Basics/Question12.py
# Print Fibonacci Series

n = eval(input("Enter the number: "))
fValue = 0; sValue = 1; tValue = -1
print(fValue, sValue, end = ' ')

for i in range(1, n - 1):
    tValue = fValue + sValue
    fValue = sValue
    sValue = tValue

    print(tValue, end = " ")

File: 3. DSA/02. C++/01. Basics/Question12.cpp
// Print Fibonacci Series

#include <iostream>
using namespace std;

int main()
{
    int num;
    cout << "Enter num: ";
    cin >> num;

    int fValue = 0, sValue = 1, tValue;
    // fValue = firstValue, sValue = secondValue, tValue = thirdValue

    if(num == 1)
    {
        cout << endl << fValue;
        return 0;
    }

    cout << endl << fValue << " " << sValue << " ";
    for(int i = 3; i <= num; i++)
    {
        tValue = fValue + sValue;
        fValue = sValue;
        sValue = tValue;

        cout << tValue << " ";
    }

    cout << endl;
}